docs/external_system.md

```diff
@@ -250,110 +275,70 @@ steps needed to configure SAT to use externally-accessible API endpoints exposed
-|`sat nid2xname`|Requires authentication to the API gateway.|`sat-nid2xname`|Translate node IDs to node XNames.|
-|`sat sensors`|Requires authentication to the API gateway.|`sat-sensors`|Report current sensor data.|
-|`sat setrev`|Requires S3 to be configured for site information such as system name, serial number, install date, and site name.|`sat-setrev`|Set HPE Cray EX system revision information.|
-|`sat showrev`|Requires API gateway authentication in order to query the Interconnect from HSM. Requires S3 to be configured for site information such as system name, serial number, install date, and site name.|`sat-showrev`|Print revision information for the HPE Cray EX system.|
-|`sat slscheck`|Requires authentication to the API gateway.|`sat-slscheck`|Perform a cross-check between SLS and HSM.|
-|`sat status`|Requires authentication to the API gateway.|`sat-status`|Report node status across the HPE Cray EX system.|
-|`sat swap`|Requires authentication to the API gateway.|`sat-swap`|Prepare HSN switch or cable for replacement and bring HSN switch or cable into service.|
-|`sat xname2nid`|Requires authentication to the API gateway.|`sat-xname2nid`|Translate node and node BMC XNames to node IDs.|
-|`sat switch`|**This command has been deprecated.** It has been replaced by `sat swap`.|
-
-In order to authenticate to the API gateway, you must run the `sat auth` command. This command will prompt for a password
-on the command line. The username value is obtained from the following locations, in order of higher precedence to lower
-precedence:
-
-- The `--username` global command-line option.
-- The `username` option in the `api_gateway` section of the config file at `~/.config/sat/sat.toml`.
-- The name of currently logged in user running the `sat` command.
-
-If credentials are entered correctly when prompted by `sat auth`, a token file will be obtained and saved to
-`~/.config/sat/tokens`. Subsequent sat commands will determine the username the same way as `sat auth` described above,
-and will use the token for that username if it has been obtained and saved by `sat auth`.
+Initially, as part of the installation and configuration, SAT authentication is
+set up so SAT commands can be used in later steps of the install process. The
+admin account used to authenticate with `sat auth` must be enabled in Keycloak
+and must have its *assigned role* set to *admin*. For more information on
+editing *Role Mappings*, see *Create Internal User Accounts in the Keycloak
+Shasta Realm* in the [Cray System Management Documentation](https://cray-hpe.github.io/docs-csm/).
+For more information on authentication types and authentication credentials,
+see [SAT Command Authentication](introduction.md#sat-command-authentication).
 
 ### Prerequisites
 
-- The `sat` CLI has been installed following [Install and Configure SAT](#install-and-configure-sat).
+- The `sat` CLI has been installed following [Install and Configure
+  SAT](#install-and-configure-sat).
```

Comment:
I would remove this prerequisite here. This is an alternate external-system install procedure, and it does not depend on installing and configuring SAT as described in the linked document.

Reply:
Removed in next commit